tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
This is to adopt the ESM architecture in the @excalidraw package. I chose `bundler` as this project uses vite which is typically a go-to option in vite project

Reply:
Should be fine.

Reply:
But, CI is failing.

Reply:
Do you have time to look into it?

Reply:
Yes, it's been a bit of struggle with vitest config. Should be able to put something up in a few days